service-automation/config.properties

```diff
@@ -40,6 +43,7 @@ Student.url=/api/students/
 
 #REQUEST PAYLOADS
 ################################
+staff.create= {"firstName":"$firstname","idNumber":"$id","gender":"$gender","staffDesignation":{"version":0,"name":"$name","description":"$description","id":"4859447556082907034"},"department":{"id":1,"departmentName":"Literature","createdDate":1519154497000,"createdBy":"ADMIN","updatedDate":1519154497000,"updatedBy":"ADMIN"},"academicYear":{"id":1,"academicYearName":"2019-20","academicYearState":"CURRENT","createdDate":1580840789000,"createdBy":"ADMIN","updatedDate":1584439733000,"updatedBy":"ADMIN","startDate":"18-03-2019","endDate":"15-03-2020"},"residentialAddress":{"addressLine1":"$addressline1","addressLine2":"","city":"$city","state":"$state","country":"$country","pincode":"$pincode"},"perminantAddress":{"addressLine1":"wuie","addressLine2":"","state":"$state","city":"hjjl","country":"lkjlj","pincode":"8890"},"primaryPhoneNumber":"$primaryPhoneNumber","primaryEmailId":"$primaryEmailId","staffAllotments":[{"branchStandardSection":{"id":56,"branch":{"id":4,"version":0,"name":"HYD Branch","description":"HYD Branch","branchAddressDetails":{"id":417,"addressLine1":"kondapur","addressLine2":"kondapur","addressLine3":null,"state":"Telangana","city":"Hyderabad","country":"India","pincode":"700050"},"emailId":"khusbu@appyschool.in","contactNumber1":"9774572449","contactNumber2":"9337689667","createdDate":null,"createdBy":null,"updatedDate":null,"updatedBy":null},"standard":{"id":36,"version":2,"name":"BTECH","description":"Pharma","standardLevel":"PRIMARY","subjectDetails":[{"id":5,"code":null,"name":"Science","orderRank":null,"description":"Science","createdDate":1519116731000,"createdBy":"ADMIN","updatedDate":1519116731000,"updatedBy":"ADMIN"}],"createdDate":null,"createdBy":null,"updatedDate":null,"updatedBy":null},"section":{"id":1,"version":1,"name":"A","description":"Section A","createdDate":null,"createdBy":null,"updatedDate":null,"updatedBy":null},"createdDate":1591450080000,"createdBy":"ADMIN","updatedDate":1591450080000,"updatedBy":"ADMIN"},"subjectDetails":{"id":5,"code":null,"name":"Science","orderRank":null,"description":"Science","createdDate":1519116731000,"createdBy":"ADMIN","updatedDate":1519116731000,"updatedBy":"ADMIN"}}]}
```

Comment:
Values should not be hardcoded. I see that designation, academic year, department, etc are hardcoded.